problems/[230619] Top K Frequent Words.js

```diff
@@ -13,7 +14,6 @@ const topKFrequent = function (words, k) {
     frequency[word] =
       frequency[word] === undefined ? 1 : (frequency[word] += 1);
   });
-
   Object.entries(frequency)
     .sort()
     .sort((a, b) => b[1] - a[1])
```

Comment:
```suggestion
return Object.entries(frequency)
    .sort((a, b) => {
      if (a[1] !== b[1]) {
        return b[1] - a[1];
      } else {
        return a[0] > b[0] ? 1 : -1;
      }
    })
    .slice(0, k)
    .map((word) => word[0]);
```
이렇게 작성하면 `sort()` 메서드를 한번만 작성해서 풀 수 있습니다. 
이것도 현정님이 피드백 주셨슴당..ㅎㅎ